.github/workflows/pre-main.yaml

```diff
@@ -158,17 +158,17 @@ jobs:
       PFLT_DOCKERCONFIG: '/home/runner/.docker/config'
 
     steps:
-      - name: Free Disk Space (Ubuntu)
-        uses: jlumbroso/free-disk-space@main
-        with:
-          tool-cache: false
-          large-packages: true
-          android: true
-          dotnet: true
-          haskell: true
-          docker-images: true
-          swap-storage: true
-        continue-on-error: true
+      # - name: Free Disk Space (Ubuntu)
```

Comment:
We have a Git history; there is no need for commented-out code.

Reply:
Yeah I know I was mostly just testing.  Seems like we're still on the old runners yet:
`/var/lib/containerd/io.containerd.content.v1.content/ingest/e7f5ce25d5889a54f1a5d43ee7f77067a6bdd2544472124edcbca557ac10f40f/data: no space left on device: unknown`

Going to mark WIP and try again tomorrow.